vissl/hooks/log_hooks.py

```diff
@@ -207,7 +217,10 @@ def on_update(self, task: "tasks.ClassyTask") -> None:
                         f"btime({self.btime_freq}iters): {rolling_btime} ms; "
                         f"rolling_eta: {rolling_eta_str}"
                     )
+                    log_data["btime_iters"] = rolling_btime
+                    log_data["rolling_eta"] = rolling_eta_str
                 logging.info(log_str)
```

Comment:
let's replace this line with `logging.info(json.dumps(log_data))` and completely get rid of `log_str`